cli/cmd/devnet.go

```diff
@@ -145,7 +158,7 @@ func fund(ctx context.Context, address string, rpcURL string) error {
 	bf, _ := b.Float64()
 	bf /= params.Ether
 
-	log.Info(ctx, "Account funded", "address", address, "balance", fmt.Sprintf("%.4f ETH", bf))
+	log.Info(ctx, "Account funded", "address", cfg.Address, "balance", fmt.Sprintf("%.4f ETH", bf))
```

Comment:
Feel free to ignore this suggestion as it is purely subjective

```suggestion
	log.Info(ctx, "Account funded", "address", cfg.Address, "balance", fmt.Sprintf("%.2f ETH", bf))
```